composer.json

```diff
@@ -34,11 +34,14 @@
             "tests/Drivers/BaseDriverTestCase.php"
         ]
     },
+    "conflict": {
+        "mnapoli/silly": ">=1.8.1 <1.8.3"
+    },
```

Comment:
Do we really need the conflict here? Can't we bump the minimum version?

Reply:
We could. Would `^1.8.3` be sufficient?

Reply:
Unfortunately `^1.7` would be the very minimum matching set of requirements for compatibility with Valet in its present state, without triggering errors. 
Actually, `^1.5` is safest, for compatibility with older PHP that may be present on a given Mac.

Once we completely drop any backwards compatibility pre-7.4, then we could bump to `1.8.3`.

Reply:
Makes sense. So, we will keep the `^1.5` requirement that I had bumped. Do I remove the conflict entry?

Reply:
I suggest leaving it as-is, but it's not my call. Deferring to Dries ;)

Reply:
The conflict entry is there to avoid accidentally installing 1.8.1 or 1.8.2—or not updating from those two versions—in order for this `--isolate` PR to work.

Reply:
The conflict is fine. No worries. Ideally I want to avoid those as much as possible but nbd.